### PR TITLE
Adding .fog.example and amending docs

### PR DESCRIPTION
**Pull request by palladius**
[trying my first pull request here! So apologies for newbbisities]

Adding a sample file for .fog and proposing in the docs to just append this sample into your home fog (I'm not an expert but I presume this is safer than a CP as people might have stuff already in there). Ideally if you like this idea I can actually remove the same stuff from README (more DRY) - but I wasnt sure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ my_credential:
     google_json_key_location: /path/to/my-project-xxxxxxxxxxxx.json
 ```
 
+Or simply `cat .fog.example >> ~.fog` and edit parameters in the file.
```
Comment (Temikus) [inline README.md]: `~.fog` is an incorrect glob, as it'll resolve to: /Users/myuser.fog, you need a slash: `~/.fog`

File: .fog.example [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#####################################################
+# This is configuration snippet to configure Google Cloud Platform
+# for fig library.
```
Comment (Temikus) [inline .fog.example]: typo

File: .fog.example [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+#
+# 1. Copy this file into your home dir: "~/.fog"
+# 2. Follow instructions to generate a private key: 
+#   https://cloud.google.com/storage/docs/authentication#generating-a-private-key
+
+my_credential:
```
Comment (Temikus) [inline .fog.example]: Although I do admit that's our doc issue, I'm not sure if `my_credential` will actually work by default if you didn't explicitly load them via Fog::Credentials. AFAIK only "default" works by default and whatever is in "test" is loaded by tests. 

Have you tested this?

**Comment by palladius** [comment]
ACK on comments - thanks.

1. Now I don't know how to amend this pull request to address the required changes. Let me study this out :) Using https://gist.github.com/Chaser324/ce0505fbed06b947d962 as study point.

2. Ok - addressed both bugs in new public branch: https://github.com/palladius/fog-google/tree/newfeature

**Comment by palladius** [comment]
New commit which addresses Temikus comments: https://github.com/palladius/fog-google/commit/93781eb72386c17866c3cbc52726b87f7b2ea14f

**Comment by Temikus** [comment]
@palladius LGTM, thanks for contributing \o/ Sorry it took a while.

One last note - I suggest setting your git config to a proper email, so you get the contribution tracked on github properly. Currently your commit's email is set to "palladiusbonton+derek.zrh.corp.google.com@gmail.com". Either associate that with GitHub or:
```
git rebase -i HEAD~2
git commit --amend --author="Riccardo Carlesso email-tied-to-github@email.com"
```

Cheers!

**Comment by icco** [comment]
(You'll need to `git push -f` after rebasing for this to update.) 👋 @palladius!

**Comment by palladius** [comment]
Thanks bros! Done the following:

git rebase -i HEAD~2
git commit --amend --author="Riccardo Carlesso <riccardo.carlesso@gmail.com>"
git push -f

**Comment by icco** [comment]
Thanks!

**Pull request merged**